### PR TITLE
add storage type and s3 endpoint on helm chart

### PR DESCRIPTION
**Pull request by drgarcia1986**
Specific stuff to works fine with minio as slug storage.

---
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/264)

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -73,7 +73,10 @@ Parameter | Description | Default
 `db.hostname`| (Optional) Database hostname, if defined use mysql instead of sqlite| `""`
 `db.username` | (Optional) Database username | `""`
 `db.password` | (Optional) Database password | `""`
+`storage.type` | Type of storage | `s3`
+`aws.s3.force_path_style` | To force path style instead of subdomain-style | `false`
 `aws.s3.bucket` | S3 bucket path | `""`
+`aws.s3.endpoint` | AWS Endpoint | `""`
```
Comment (yagonobre) [inline helm/README.md]: I think that it's optional

File: helm/chart/teresa/templates/deployment.yaml [hunk 2]
```diff
@@ -44,8 +44,14 @@ spec:
           value: /etc/teresa-keys/teresa.rsa
         - name: TERESA_SECRETS_PUBLIC_KEY
           value: /etc/teresa-keys/teresa.rsa.pub
+        - name: TERESA_STORAGE_TYPE
+          value: {{ .Values.storage.type }}
+        - name: TERESA_STORAGE_AWS_ENDPOINT
```
Comment (yagonobre) [inline helm/chart/teresa/templates/deployment.yaml]: only append this if the `.Values.aws.s3.endpoint` was provide

File: helm/README.md (hunk 1 above)
Comment (drgarcia1986) [inline helm/README.md]: you're right!

**Comment by drgarcia1986** [comment]
updated @yagonobre !
could you check again?

**Pull request merged**